src/docs/product/integrations/integration-platform/ui-components/formfield.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: FormField Components
+sidebar_order: 4
+description: "Learn about FormField components."
+---
+
+The issue linking and alert rule action UI components can utilize `FormField` components to display attributes in their respective modals. A `FormField` can be one of three types (which behave like their HTML equivalents) — `Text`, `TextArea,` and `Select`.
+
+## Text
+
+### Schema
+
+```javascript
+{
+    "type": "text",
+    "label": <String>,
+    "name": <String>,
+    (optional) default: <String>,
+}
+```
+
+### Attributes
+
+- `label` - (Required) Label text to be rendered for the form field
+- `name` - (Required) Value to use in `name` attribute of the field
+- `default` - default to pre-populate with. Options include `issue.title` and `issue.description`
```

Comment:
Idk what it should look like but this needs a note that it only applies to Issue linking